rules/slowloris.go

```diff
@@ -35,7 +35,7 @@ func containsReadHeaderTimeout(node *ast.CompositeLit) bool {
 	for _, elt := range node.Elts {
 		if kv, ok := elt.(*ast.KeyValueExpr); ok {
 			if ident, ok := kv.Key.(*ast.Ident); ok {
-				if ident.Name == "ReadHeaderTimeout" {
+				if ident.Name == "ReadHeaderTimeout" || ident.Name == "ReadTimeout" {
```

Comment:
Wouldn't that mean that a configuration where the programmer sets `ReadHeaderTimeout` but not `ReadTimeout` would be considered valid?  Because it seems like that would still allow the remote peer to slow down the server by sending headers quickly and then not sending the body.

That's technically not an error, since the documentation of `ReadHeaderTimeout` implies that that is precisely the intention behind this parameter, but I feel like a tool like `gosec` should at least warn the programmer about that.

Reply:
I think that such an attack is technically more difficult to perform that an attack without any timeout. gosec warns the programmer to setup some timeout. I think is fair enough in this case.